client/src/api/index.js

```diff
@@ -32,8 +32,7 @@ export const findOrCreateTheme = (themeData) => api.post('/theme', themeData);
 export const addWebPage = (teamId, pageName) => api.post(`/clientWebsite/${teamId}/add_page`, pageName);
 export const getWebsiteInfo = (teamId) => api.get(`/clientWebsite/${teamId}`);
 export const deleteWebPage = (teamId, pageName) => api.post(`/clientWebsite/${teamId}/delete_page`, pageName);
-export const updatePublicationOptions = (teamId, options) => api.post(`/clientWebsite/${teamId}/updatePublicationOptions`, options);
-export const updateWebsiteTitle = (teamId, title) => api.post(`/clientWebsite/${teamId}/updateTitle`, title);
+export const updateClientWebMetadata = (teamId, changes) => api.patch(`/clientWebsite/${teamId}`, changes);
```

Comment:
do we still want to refer as client web metadata, or change it to team site metadata to ensure consistency across?

Reply:
All the endpoints currently use `clientWebsite` and there's some other places in Researchify Code that refers to 'client' instead of 'team'. So I'm thinking of leaving it for now, else I'll need to change those as well (at least the endpoints) for consistency and redo all the testing.